.cirrus.yml

```diff
@@ -94,8 +94,11 @@ report_function_task:
             BUILD_NAME: ${UBUNTU_PRIOR_NAME}
             VM_IMAGE_NAME: ${UBUNTU_PRIOR_IMAGE_NAME}
     # Ubuntu has sos installed by default, so remove that to avoid any conflicts
-    setup_script: |
+    remove_sos_script: |
         if [ $(command -v apt) ] && [ $(command -v sosreport) ]; then
             apt -y purge sosreport
+            apt update
+            apt -y install python3-pip
         fi
-    main_script: ./tests/simple.sh
+    setup_script: pip3 install avocado-framework==84.0
```

Comment:
84 is the current avocado LTS version.

Reply:
It seems relatively stable to me - I guess I'd prefer to do their "regular" versions to get features/fixes faster.  We can always stop on the next LTS.

Reply:
> It seems relatively stable to me - I guess I'd prefer to do their "regular" versions to get features/fixes faster. We can always stop on the next LTS.

Avocado tries very hard to follow a deprecate, document, and then exclude/change/replace features, even for non LTS releases.  Before every release, we test how the latest Avocado would behave if enabled in some of the projects that depend on it.  For example this is checking how the latest cut before Avocado 87.0 would behave on QEMU:

https://gitlab.com/cleber.gnu/qemu/-/pipelines/283543999

Also please notice that this specific release has at least a new feature and a bugfix inspired by sos:

* https://github.com/avocado-framework/avocado/pull/4467
* https://github.com/avocado-framework/avocado/pull/4490

So I tend to agree that you would benefit from using the non-LTS version to pickup those (and future enhancements).  It may be that after a few development cycles for the tests, you stabilize and then end up settling on a future LTS version.

Anyway, just my 2 cents.